modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/portlet/action/GetObjectFieldInfoMVCResourceCommand.java

```diff
@@ -72,6 +75,30 @@ protected void doServeResource(
 		JSONPortletResponseUtil.writeJSON(
 			resourceRequest, resourceResponse,
 			JSONUtil.put(
+				"listTypeDefinitionsURL",
+				() -> {
+					if (StringUtil.equals(
+							objectField.getBusinessType(),
+							ObjectFieldConstants.BUSINESS_TYPE_PICKLIST)) {
+
+						RequestBackedPortletURLFactory
+							requestBackedPortletURLFactory =
+								RequestBackedPortletURLFactoryUtil.create(
+									resourceRequest);
+
+						return PortletURLBuilder.create(
+							requestBackedPortletURLFactory.
+								createControlPanelRenderURL(
```

Comment:
Why did you choose the method `createControlPanelRenderURL()`?
Can you point me the pattern that you based your work?

I see that this class has a method `requestBackedPortletURLFactory.createRenderURL()`. Do you know the difference between these methods?

Reply:
Hi @marcelabc, the difference between these method is very simple. The `createControlPanelRenderURL()` shows the Portlet name and the open menu. The `createRenderURL()` doesn't shows the Portlet name and the open menu.

Using the `createControlPanelRenderURL()`:
![image](https://github.com/liferay-objects/liferay-portal/assets/30407820/dedc3e74-fb73-45cc-a2a6-4d976a085f86)

Using the `createRenderURL()`:
![image](https://github.com/liferay-objects/liferay-portal/assets/30407820/c830052e-b412-4225-97f5-7a285e19d9b5)

Reply:
Thank you for the explanation!